buildpack.md

```diff
@@ -744,31 +749,39 @@ The stack ID:
 
 The stack `build-images` and `run-images` are suggested sources of images for platforms that are unaware of the stack ID. Buildpack authors MUST ensure that these images include all mixins specified in `mixins`.
 
-### launch.toml (TOML)
+### app.toml (TOML)
 
 ```toml
 [[processes]]
 type = "<process type>"
 command = "<command>"
+
+[[slices]]
+paths = ["<app sub-path glob>"]
 ```
 
-Buildpacks MUST specify:
+The buildpack MAY specify any number of processes or slices.
+
+For each process, the buildpack MUST specify:
 
-- A unique process type for each entry within a `launch.toml` file.
+- A unique process type for each entry within a `app.toml` file.
 - A command that is valid when executed using the Bash 3+ shell.
 
-### develop.toml (TOML)
+For each slice, buildpacks MUST specify zero or more path globs such that each path is either:
 
-```toml
-[[processes]]
-type = "<process type>"
-command = "<command>"
-```
+- Relative to the root of the app directory without traversing outside of the app directory.
+- Absolute and contained within the app directory.
 
-Buildpacks MUST specify:
+Path globs MUST:
 
-- A unique process type for each entry within a `develop.toml` file.
-- A command that is valid when executed using the Bash 3+ shell.
+- Follow the pattern syntax defined in the [Go standard library](https://golang.org/pkg/path/filepath/#Match).
+- Match zero or more files or directories.
+
+The lifecycle MUST process each slice as if all files matched in preceding slices no longer exists in the app directory.
+
+The lifecycle MUST accept slices that do not contain any files or directory. However, the lifecycle MAY warn about such slices.
+
+The lifecycle MUST include all unmatched files in the app directory in zero or more additional layers in the OCI image.
```

Comment:
what does "zero or more additional layers" mean here?

Reply:
Just means that if there are zero unmatched files, it isn't necessary to include any additional layers.

Reply:
Clarified in https://github.com/buildpack/spec/pull/36/commits/a87bf21c66b5d8992e4fe37761a1e96bab1cde9b.